app/javascript/src/Navigation/components/header/AccountSettingsMenu.jsx

```diff
@@ -1,6 +1,9 @@
-import React from 'react'
+// @flow
 
-const AccountSettingsMenu = ({accountSettingsLink, accountSettingsClass}) => (
+import * as React from 'react'
+import { AccountSettingsProps } from 'Navigation/components/header/types'
```

Comment:
I wouldn't put Prop and State type definitions in other files, even if it were used elsewhere but it helps encapsulation.